docs/content/examples/tutorials/basic-installation.md

```diff
@@ -84,142 +79,3 @@ services:
-
-1. If you're running a version of `docker-mailserver` earlier than v10.2, [you'll need to get `setup.sh`][docs-setup-script]. Otherwise you can substitute `./setup.sh <command>` with `docker exec mailserver setup <command>`.
-
-2. Pull the docker image: `docker pull docker.io/mailserver/docker-mailserver:latest`.
-
-3. Create the file `docker-compose.yml` with a content like this:
-
-    !!! example
-
-        ```yaml
-        services:
-          mailserver:
-            image: docker.io/mailserver/docker-mailserver:latest
-            container_name: mailserver
-            # Provide the FQDN of your mail server here (Your DNS MX record should point to this value)
-            hostname: mail.example.com
-            ports:
-              - "25:25"
-              - "587:587"
-              - "465:465"
-            volumes:
-              - ./docker-data/dms/mail-data/:/var/mail/
-              - ./docker-data/dms/mail-state/:/var/mail-state/
-              - ./docker-data/dms/mail-logs/:/var/log/mail/
-              - ./docker-data/dms/config/:/tmp/docker-mailserver/
-              - /etc/localtime:/etc/localtime:ro
-            environment:
-              - ENABLE_FAIL2BAN=1
-              # Using letsencrypt for SSL/TLS certificates
-              - SSL_TYPE=letsencrypt
-              # Allow sending emails from other docker containers
-              # Beware creating an Open Relay: https://docker-mailserver.github.io/docker-mailserver/latest/config/environment/#permit_docker
-              - PERMIT_DOCKER=network
-              # You may want to enable this: https://docker-mailserver.github.io/docker-mailserver/latest/config/environment/#spoof_protection
-              # See step 8 below, which demonstrates setup with enabled/disabled SPOOF_PROTECTION:
-              - SPOOF_PROTECTION=0
-            cap_add:
-              - NET_ADMIN # For Fail2Ban to work
-            restart: always
-        ```
-
-    - The docs have a detailed page on [Environment Variables][docs-environment] for reference.
-
-    !!! note "Firewalled ports"
-
-        You may need to open ports `25`, `587` and `465` on the firewall. For example, with the firewall `ufw`, run:
-
-        ```sh
-        ufw allow 25
-        ufw allow 587
-        ufw allow 465
-        ```
-
-4. Configure your DNS service to use an MX record for the _hostname_ (eg: `mail`) you configured in the previous step and add the [SPF][docs-spf] TXT record.
-
-    If you manually manage the DNS zone file for the domain, it would look something like this:
-
-    ```txt
-    mail      IN  A   10.11.12.13
-
-    ; mail-server for example.com
-        3600  IN  MX  1  mail.example.com.
-
-    ; Add SPF record
-              IN TXT "v=spf1 mx ~all"
-    ```
-
-    Then don't forget to change the serial number and to restart the service.
```

Comment:
I am currently rewriting DKIM/DMARC/SPF docs and the DNS setup there would be more precise. I'd argue that the current state in "Usage" is fine though, because it is mostly about MX/A/PTR, which is not too complicated (at least definitely not as complicated as DKIM/DMARC/SPF records). A PR I have in the works will resolve this concern anyway though :)